Bottom: 71ef58a3dc2fb4dd2a392e7f4587d76e09e077ca
Top:    9c2843f13567e3e5b9b87674f3a0d5613aa57fb7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-26 23:52:16 +0000

Refresh of hid-gerber-remove-unused-vaira

---

diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index 27496bf..8f04180 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -443,7 +443,6 @@ gerber_do_export (HID_Attr_Val * options)
   static int saved_layer_stack[MAX_LAYER];
   int save_ons[MAX_LAYER + 2];
   FlagType save_thindraw;
-  bool absolute;
 
   save_thindraw = PCB->Flags;
   CLEAR_FLAG(THINDRAWFLAG, PCB);
